ios/BT/ExposureManager.swift

```diff
@@ -337,9 +337,10 @@ final class ExposureManager: NSObject {
       var newExposures: [Exposure] = []
       if let summary = exposureSummary {
         summary.daySummaries.forEach { (daySummary) in
-          if daySummary.isAboveScoreThreshold(with: exposureConfiguraton) {
+          if daySummary.isAboveScoreThreshold(with: exposureConfiguraton) &&
+              self.btSecureStorage.canStoreExposure(for: daySummary.date) {
             let exposure = Exposure(id: UUID().uuidString,
-                                    date: daySummary.date.posixRepresentation)
+                                    date: daySummary.date.toMidnight.posixRepresentation)
```

Comment:
I don't think we should do toMidnight here? I believe the dates are already normalized to be by day?

Reply:
then wouldn't the time of day have no effect? The goal is for each exposure to be saved to the database at the same time of day

Reply:
let me know if there's a different way to accommodate this: https://github.com/Path-Check/gaen-mobile/pull/370#discussion_r487446174

Reply:
The documentation doesn't specify with sharp clarity, but just like the previous date fields were rounded by day, I assume this one is too. Let me double check but I'm fairly certain that this already is normalized by day. Rounding the date to start of day here (possibly in a different time zone from how it was rounded in the first place) is what leads to bugs like the one we had before where we couldn't tell what exact day an exposure happened on.